dev-docs/Cloud-Env-Testing.md

```diff
@@ -121,3 +121,36 @@ Before running the script, ensure that:
 
 **Note**: The script will ask for confirmation before deleting each environment, unless you set the `interactive` flag
 to `false`.
+
+### Delete Environment Workflow
```

Comment:
I think this part should go first (before the script info)

Reply:
Updated